bigquery_etl/cli/query.py

```diff
@@ -918,24 +941,78 @@ def _run_query(
         if "query" not in query_arguments:
             query_arguments = ["query"] + query_arguments
 
+        query_text = render_template(
+            query_file.name,
+            template_folder=str(query_file.parent),
+            templates_dir="",
+            format=False,
+            **addl_templates,
+        )
+
+        # create a session, setting default project and dataset
```

Comment:
Flyby comment: there's ample discussion in this PR, but a comment explaining why the query is run in a session in this case could be useful in future